app/src/androidTest/java/com/android/sample/ui/activity/CreateActivityScreenTest.kt

```diff
@@ -1,99 +1,197 @@
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputTitleCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputTitleCreate").assertIsDisplayed()
+  }
 
-    @Test
-    fun createActivityScreen_displaysDescriptionField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
-        }
-        composeTestRule.onNodeWithTag("inputDescriptionCreate").assertExists()
-        composeTestRule.onNodeWithTag("inputDescriptionCreate").assertIsDisplayed()
+  @Test
+  fun createActivityScreen_displaysDescriptionField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputDescriptionCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputDescriptionCreate").assertIsDisplayed()
+  }
 
-    }
+  @Test
+  fun createActivityScreen_displaysDateField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputDateCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputDateCreate").assertIsDisplayed()
+  }
 
-    @Test
-    fun createActivityScreen_displaysDateField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
-        }
-        composeTestRule.onNodeWithTag("inputDateCreate").assertExists()
-        composeTestRule.onNodeWithTag("inputDateCreate").assertIsDisplayed()
+  @Test
+  fun createActivityScreen_displaysPriceField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputPriceCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputPriceCreate").assertIsDisplayed()
+  }
 
-    }
+  @Test
+  fun createActivityScreen_displaysPlacesLeftField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputPlacesCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputPlacesCreate").assertIsDisplayed()
+  }
+
+  @Test
+  fun createActivityScreen_displaysLocationField() {
+    composeTestRule.setContent { CreateActivityScreen(mockViewModel, mockNavigationActions) }
+    composeTestRule.onNodeWithTag("inputLocationCreate").assertExists()
+    composeTestRule.onNodeWithTag("inputLocationCreate").assertIsDisplayed()
+  }
+
+  @Test
+  fun attendeeRowDisplaysCorrectInformation() {
+    val attendee = SimpleUser("John", "Doe", 30)
+    val attendees = listOf(attendee)
 
-    @Test
-    fun createActivityScreen_displaysPriceField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
+    composeTestRule.setContent {
+      Row(
+          modifier = Modifier.padding(8.dp).background(Color(0xFFFFFFFF)),
+      ) {
+        Text(
```

Comment:
I do not get why you are coding some UI here given the signature of the function. You could edit this part or remove it.